homeassistant/components/traccar_server/config_flow.py

```diff
@@ -161,7 +161,7 @@ async def async_step_user(
         )
 
     async def async_step_import(self, import_data: dict[str, Any]) -> ConfigFlowResult:
-        """Import an entry."""
+        """Import an entry from `tracar` domain (see #109226)."""
```

Comment:
This should be removed not changed, it has gone past 6 months.

Reply:
I'm not sure of the process here, as it's a cross-domain/cross-integration migration.

Reply:
I had a quick look at the code for `traccar`, and I think it's better if the cleanup is done by the code-owner.
I'm really not sure what can and can't be removed there...

Reply:
Should be addressed in #125763